integration-test/Css.ts

```diff
@@ -119,25 +119,25 @@ class CssBuilder<T extends Properties1> {
   display(value: Properties["display"]) { return this.add("display", value); }
 
   // flexbox
-  get justifyStart() { return this.add("justifyContent", "flex-start"); }
-  get justifyEnd() { return this.add("justifyContent", "flex-end"); }
-  get justifyCenter() { return this.add("justifyContent", "center"); }
-  get justifyBetween() { return this.add("justifyContent", "space-between"); }
-  get justifyAround() { return this.add("justifyContent", "space-around"); }
-  get justifyEvenly() { return this.add("justifyContent", "space-evenly"); }
-  justify(value: Properties["justifyContent"]) { return this.add("justifyContent", value); }
-  get selfStart() { return this.add("alignSelf", "flex-start"); }
-  get selfEnd() { return this.add("alignSelf", "flex-end"); }
-  get selfCenter() { return this.add("alignSelf", "center"); }
-  get selfBaseline() { return this.add("alignSelf", "baseline"); }
-  get selfStretch() { return this.add("alignSelf", "stretch"); }
-  self(value: Properties["alignSelf"]) { return this.add("alignSelf", value); }
-  get itemsStart() { return this.add("alignItems", "flex-start"); }
-  get itemsEnd() { return this.add("alignItems", "flex-end"); }
-  get itemsCenter() { return this.add("alignItems", "center"); }
-  get itemsBaseline() { return this.add("alignItems", "baseline"); }
-  get itemsStretch() { return this.add("alignItems", "stretch"); }
-  items(value: Properties["alignItems"]) { return this.add("alignItems", value); }
+  get js() { return this.add("justifyContent", "flex-start"); }
+  get je() { return this.add("justifyContent", "flex-end"); }
+  get jc() { return this.add("justifyContent", "center"); }
+  get jb() { return this.add("justifyContent", "space-between"); }
+  get ja() { return this.add("justifyContent", "space-around"); }
+  get jEvenly() { return this.add("justifyContent", "space-evenly"); }
+  j(value: Properties["justifyContent"]) { return this.add("justifyContent", value); }
+  get asStart() { return this.add("alignSelf", "flex-start"); }
+  get ase() { return this.add("alignSelf", "flex-end"); }
+  get asc() { return this.add("alignSelf", "center"); }
+  get asb() { return this.add("alignSelf", "baseline"); }
+  get asStretch() { return this.add("alignSelf", "stretch"); }
+  as(value: Properties["alignSelf"]) { return this.add("alignSelf", value); }
+  get ais() { return this.add("alignItems", "flex-start"); }
+  get aie() { return this.add("alignItems", "flex-end"); }
+  get aic() { return this.add("alignItems", "center"); }
+  get aib() { return this.add("alignItems", "baseline"); }
+  get aiStretch() { return this.add("alignItems", "stretch"); }
```

Comment:
`Align Items Stretch` --> `ais` and `Align Items flex Start` --> `ais` overlap, so I made this one `aiStretch` ... but maybe `ais` should be `aifs` i.e. `Align Items Flex Start`? And then `aie` -> `aife`?

Reply:
I'm good with a couple more letters in it, so `aife`/`aifs`. Much more concise than what we have today, and hopefully adding the extra character will reduce confusion.

Reply:
Agreed, pushed up...